pkg/controllermanager/controller/shoot/shoot_maintenance_control.go

```diff
@@ -109,7 +111,7 @@ func (c *Controller) shootMaintenanceRequeue(key string, shoot *gardencorev1beta
 		nextMaintenance = time.Now().Add(duration)
 	)
 
-	logger.Logger.Infof("[SHOOT MAINTENANCE] %s - Scheduled maintenance in %s at %s", key, duration, nextMaintenance.UTC())
+	logger.Logger.Infof("[SHOOT MAINTENANCE] %s - Scheduled maintenance in %s at %s", key, duration.Round(time.Minute), nextMaintenance.UTC().Round(time.Minute))
```

Comment:
Can you elaborate on why `Round()` is needed/used here?

Reply:
This is just to make it more readable. I think having the next maintenance time rounded to the second in the logs is sufficient (no need for nano seconds etc.). This is only the logs and will not affect the actual scheduled maintenance.